Pseudocode/Dynamic Programming/Bursting Balloons/Source Code.tex

```diff
@@ -0,0 +1,71 @@
+% Set the Page Layout
+\documentclass[12pt]{article}
+\usepackage[inner = 2.0cm, outer = 2.0cm, top = 2.0cm, bottom = 2.0cm]{geometry}
+
+
+% Package to write pseudo-codes
+\usepackage{algorithm}
+
+% Remove the 'end' at the end of the algorithm
+\usepackage[noend]{algpseudocode}
```

Comment:
Since there are 2 functions involved, I'd rather retain the **end** for some functions procedures. See the Gist to know how to suppress the **end** in some while retaining them in others.